app/helpers/pledges_helper.rb

```diff
@@ -16,6 +16,22 @@ def cancel_pledge_button
     end
   end
 
+  def mark_complete_button
+    content_tag :span, class: 'btn btn-primary btn-lg submit-btn btn-block',
+                       aria: { hidden: true },
+                       id: 'submit-pledge-button' do
+      t('patient.menu.mark_complete')
+    end 
+  end
+
+  def mark_incomplete_button
+    content_tag :span, class: 'btn btn-warning btn-lg cancel-btn btn-block',
+                       aria: { hidden: true },
+                       id: 'submit-pledge-button' do
+      t('patient.menu.mark_incomplete')
+    end 
+  end
+
```

Comment:
These two functions look _extremely_ similar to the submit pledge button pair (above). Thoughts on condensing, or fine to keep separate?

Reply:
I'd say keep separate - reasonable for them to diverge a bit here